### PR TITLE
Packaging / setup tweaks

### PR DESCRIPTION
**Pull request by joolswills**
setup - show install origin in manage package sections for overview
 * useful to get an overview - used bold markup but shows as grey from framebuffer - but is easier to read when differently styled imho

Wanted to get an opinion on this before I commit.

Also added a dependency section to setup also as now packages remember how they were installed it is useful to be able to switch between binary / source.

### CODE REVIEW

**Comment by joolswills** [comment]
Added the dependency section

File: scriptmodules/admin/setup.sh [hunk 1]
```diff
@@ -262,7 +262,15 @@ function package_setup() {
                 ;;
             X)
                 local text="Are you sure you want to remove $md_id?"
-                [[ "${__mod_section[$idx]}" == "core" ]] && text+="\n\nWARNING - core packages are needed for RetroPie to function!"
+                case "${__mod_section[$idx]}" in
+                    core)
+                        text+="\n\nWARNING - core packages are needed for RetroPie to function!"
+                        ;;
+                    depends)
+                        text+="\n\nWARNING - this package is required by other retropie packages - removing may cause other packages to fail."
```
Comment (cmitu) [inline scriptmodules/admin/setup.sh]: maybe capitalize `retropie` (RetroPie).

Comment (joolswills) [reply scriptmodules/admin/setup.sh]: Yeah. Sloppy text again from me.

Apart from this are you ok with it as I'll merge if so.

Comment (cmitu) [reply scriptmodules/admin/setup.sh]: yes, it looks fine.

**Pull request merged**